client/src/api/API.test.js

```
@@ -326,4 +326,52 @@ describe('Client API calls', () => {
             }
         });
     })
+
+    describe('getTeacherLectures', () => {
+        beforeEach(() => {
+            mock.reset();
+        });
+
+        const from = "2013-10-07", to = "2013-10-07";
+
+        it('returns all the teacher lectures in the db', async () => {
+            mock.onGet(base_url+"/bookings").reply(200, sample_lectures);
+            const teacherLectures = await API.getTeacherBookings();
+            for (let i = 0; i < sample_lectures.length; i++)
+                expect({deleted_at: null, ...teacherLectures[i]}).toMatchObject(sample_lectures[i]);
+        });
+
+        it('returns all the teacher lectures in a given time frame', async () => {
+            mock.onGet(base_url+"/bookings", { params: { from, to } })
+              .reply(200, sample_lectures);
+            const teacherLectures = await API.getTeacherBookings(id, from, to);
```

What is that `id` field? It is causing the test to fail cause it is undefined

---

Ok, i guess it can be removed from the test with no consequences